client/profile-wizard/steps/business-details/flows/selective-bundle/selective-extensions-bundle/index.js

```diff
@@ -0,0 +1,350 @@
+/**
+ * External dependencies
+ */
+import { useState } from '@wordpress/element';
+import {
+	Button,
+	Card,
+	CheckboxControl,
+	__experimentalText as Text,
+} from '@wordpress/components';
+import { Link } from '@woocommerce/components';
+import { __, _n, sprintf } from '@wordpress/i18n';
+import { Icon, chevronDown, chevronUp } from '@wordpress/icons';
+import interpolateComponents from 'interpolate-components';
+import { pluginNames } from '@woocommerce/data';
+import { recordEvent } from '@woocommerce/tracks';
+
+/**
+ * Internal dependencies
+ */
+import { AppIllustration } from '../app-illustration';
+import './style.scss';
+
+const generatePluginDescriptionWithLink = ( description, productName ) => {
+	return interpolateComponents( {
+		mixedString: description,
+		components: {
+			link: (
+				<Link
+					type="external"
+					target="_blank"
+					className="woocommerce-admin__business-details__selective-extensions-bundle__link"
+					href={ `https://woocommerce.com/products/${ productName }` }
+				/>
+			),
+		},
+	} );
+};
+
+const installableExtensions = [
```

Comment:
I considered moving this to the data folder, but its very specifically tied to the UI rendering of the groups of checkboxes so I've left it here for now.